op-deployer/pkg/deployer/verify/etherscan.go

```diff
@@ -42,6 +42,12 @@ func getAPIEndpoint(l1ChainID uint64) (string, error) {
 		return "https://api-sepolia.etherscan.io/api", nil // eth-sepolia
 	case 84532:
 		return "https://api-sepolia.basescan.org/api", nil // base-sepolia
+	case 10:
+		return "https://api-optimistic.etherscan.io/api", nil // optimism-mainnet
+	case 8453:
+		return "https://api.basescan.org/api", nil // base-mainnet
+	case 57073:
```

Comment:
Be nice to be able to accept this from the command line if the user wants to supply a block explorer that is not hardcoded here,